forge-gui/res/cardsfolder/v/vladimir_and_godfrey.txt

```diff
@@ -2,7 +2,7 @@ Name:Vladimir and Godfrey
 ManaCost:2 W
 Types:Legendary Creature Zombie Knight
 PT:3/2
-A:AB$ ChangeZone | Cost$ 2 W | Origin$ Graveyard | Destination$ Battlefield | Defined$ Self | ActivationZone$ Graveyard | Tapped$ True | SubAbility$ DBEffect | SpellDescription$ Rejuvenation - Return CARDNAME from your graveyard to the battlefield tapped. It perpetually gets +1/+1. Activate only if you control a 1/1 creature.
+A:AB$ ChangeZone | Cost$ 2 W | Origin$ Graveyard | Destination$ Battlefield | Defined$ Self | ActivationZone$ Graveyard | IsPresent$ Creature.YouCtrl+powerEQ1+toughnessEQ1 | PresentCompare$ EQ0 | Tapped$ True | SubAbility$ DBEffect | SpellDescription$ Rejuvenation - Return CARDNAME from your graveyard to the battlefield tapped. It perpetually gets +1/+1. Activate only if you control a 1/1 creature.
```

Comment:
EQ0 (equals 0 / false) would mean you can only activate it if you don't control one
you can just skip that param because EQ1 is default